src/components/Indexx/Cards/Cards.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "./Cards.scss";
+import Card from "./Card";
+
+const Cards = ({ characters }) => {
+  console.log(characters);
```

Comment:
Ca si c'est du debug il faut le supprimer sinon tu peux l'écrire comme ça : 
```suggestion
  console.log("characters : ", characters);
```

Reply:
Si vous êtes fan du debug par console.log jetez un oeil sur cet article (https://medium.com/@mattburgess/beyond-console-log-2400fdf4a9d8) ça va vous plaire